scalastyle-config.xml

```diff
@@ -18,59 +18,29 @@
   </parameters>
  </check>
  <check level="error" class="org.scalastyle.scalariform.EqualsHashCodeChecker" enabled="true"></check>
- <check level="error" class="org.scalastyle.scalariform.IllegalImportsChecker" enabled="true">
-  <parameters>
-   <parameter name="illegalImports"><![CDATA[sun._,java.awt._]]></parameter>
-  </parameters>
- </check>
```

Comment:
I ran into the `awt` constraint once when trying to visualize an experiment with a different Java2D library.  Do you run into the need to import `sun._` or `java.awt._` often?

Reply:
Oh, I thought it was never used so what was the point. but we could restore it

Reply:
I guess it is harmless enough, but I'm fine removing it too.